producers/zap_producer/main.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+
+	v1 "github.com/thought-machine/dracon/api/proto/v1"
+	"github.com/thought-machine/dracon/producers/zap_producer/types"
+
+	"fmt"
+	"log"
+
+	"github.com/thought-machine/dracon/producers"
+
+)
+
+func main() {
+	if err := producers.ParseFlags(); err != nil {
+		log.Fatal(err)
+	}
+
+	inFile, err := producers.ReadInFile()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var results types.ZapOut
+	if err := producers.ParseJSON(inFile, &results); err != nil {
+		log.Fatal(err)
+	}
+
+	if err := producers.WriteDraconOut(
+		"zap",
+		parseOut(&results),
+	); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func parseOut(results *types.ZapOut) []*v1.Issue {
+	issues := []*v1.Issue{}
+	for _, res := range results.Site{
+		target:= res.Name
+		for _, alert := range res.Alerts {
+			issues = append (issues, parseIssue(&alert, target))
+		}
+	}
+	return issues
+}
+
+//zap doesn't provide cvss so assigned as 0.0
+func parseIssue(r *types.ZapAlerts, target string) *v1.Issue {
+	var cvss = 0.0
```

Comment:
```suggestion
	cvss := 0.0
```

would be the idiomatic way of declaring this in Go (or you could just set the value of the `Cvss` field directly below, since this variable isn't used anywhere else).